zjit/src/codegen.rs

```diff
@@ -817,6 +818,18 @@ fn gen_fixnum_ge(asm: &mut Assembler, left: lir::Opnd, right: lir::Opnd) -> Opti
     Some(asm.csel_ge(Qtrue.into(), Qfalse.into()))
 }
 
+fn gen_anytostring(asm: &mut Assembler, val: lir::Opnd, str: lir::Opnd, state: &FrameState) -> Option<lir::Opnd> {
+    asm_comment!(asm, "call rb_obj_as_string_result");
```

Comment:
Please move the comment about calling a function down to the ccall so that it doesn't get confused with the PC-saving code